examples/dags/example_valohai_dag.py

```diff
@@ -37,3 +37,13 @@
         'batch_size': 200,
     }
 )
+
+download_model = ValohaiDownloadExecutionOutputsOperator(
+    task_id='download_model',
+    output_task_id='train_model',
+    output_path='.',
+    output_name_pattern='model.pb',
```

Comment:
Seems to me `output_name_pattern` and `model.pb` uses the special `.` character that may match a lot of undesired things?

Reply:
Indeed! I may then change that to output_name and maybe add an option is_regex that by default will be False to avoid confusion. Thanks.

Reply:
Thanks. At the end I went for for your suggestion and created output_name and output_name_pattern